src/man/sssd-ldap.5.xml

```diff
@@ -856,7 +856,7 @@
                             Certificate Authority certificates in separate
                             individual files. Typically the file names need to
                             be the hash of the certificate followed by '.0'.
-                            If available, <command>cacertdir_rehash</command>
+                            If available, <command>c_rehash</command>
```

Comment:
Hi,

maybe it would be good to mention `openssl rehash` as well. Since `c_rehash` from OpenSSL is a perl script it might not be installed by default to avoid installing the perl infrastructure.

bye,
Sumit

Reply:
You are right. c_rehash is available only when openssl-perl is installed. I'll update this PR.